### PR TITLE
Package Updater Testing PR

### PR DESCRIPTION
**Pull request by fivetran-jamie**
This pull request was created automatically 🎉:
- [ ] Ensure `.buildkite/scripts/run_models.sh` has the new run-operation line
- [ ] Update the package version (currently `v0.UPDATE.UPDATE`) in the `CHANGELOG`
- [ ] Update project versions in the `dbt_project.yml` and `integration_tests/dbt_project.yml` files
- [ ] Update Step 2 of the `README` to align either with this [source package example](https://github.com/fivetran/dbt_shopify_source/tree/main#step-2-install-the-package-skip-if-also-using-the-shopify-transformation-package) or this [transform package example](https://github.com/fivetran/dbt_shopify#step-2-install-the-package)

### CODE REVIEW

File: .buildkite/scripts/run_models.sh [hunk 1]
```diff
@@ -17,4 +17,4 @@ cd integration_tests
 dbt deps
 dbt seed --target "$db" --full-refresh
 dbt run --target "$db" --full-refresh
-dbt test --target "$db"
\ No newline at end of file
+dbt test --target "$db"dbt run-operation fivetran_utils.drop_schemas_automation --target "$db"
```
Comment (fivetran-joemarkiewicz) [inline .buildkite/scripts/run_models.sh:20]:
```suggestion
dbt test --target "$db"

dbt run-operation fivetran_utils.drop_schemas_automation --target "$db"
```

Comment (fivetran-joemarkiewicz) [reply .buildkite/scripts/run_models.sh:20]: Is there a way we can add an additional line here?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,2 +1,8 @@
+# dbt_starter_project v0.UPDATE.UPDATE
```
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md:1]: Let's also include the PR template additions here.

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -1,35 +1,25 @@
...
-**Select which warehouse(s) were used to test the PR**
-<!--- To check a warehouse remove the space and insert an x in the box (eg. [x] Bigquery). --> 
-- [ ] BigQuery
-- [ ] Redshift
-- [ ] Snowflake
-- [ ] Postgres
-- [ ] Databricks
-- [ ] Other (provide details below)
+## PR Overview
+
```
Comment (fivetran-joemarkiewicz) [inline .github/pull_request_template.md:2]:
```suggestion
```

I don't think this header is needed for the default PR checklist

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -1,35 +1,25 @@
...
+Please provide your name and company
```
Comment (fivetran-joemarkiewicz) [inline .github/pull_request_template.md:3]:
```suggestion
**Please provide your name and company**
```

Can you bold these in the default template.

**Review by fivetran-joemarkiewicz — APPROVED** [review]
This is great @fivetran-jamie!! I have just a few final adjustments to the templates and branches. Once applied this should be good to roll out to all repos!

File: .buildkite/scripts/run_models.sh (hunk 1 above)
Comment (fivetran-jamie) [reply .buildkite/scripts/run_models.sh:20]: ah yeah lemme adjust the function to insert a `\n` beforehand

File: .github/pull_request_template.md (hunk 4 above)
Comment (fivetran-jamie) [reply .github/pull_request_template.md:3]: should i bold all the questions?

Comment (fivetran-joemarkiewicz) [reply .github/pull_request_template.md:3]: Yeah that would be great. It makes for reading their responses much clearer

**Pull request closed without merging**